SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -506,11 +504,10 @@ void CScalarSolver<VariableType>::CompleteImplicitIteration(CGeometry* geometry,
       for (unsigned long iPoint = 0; iPoint < nPointDomain; iPoint++) {
         /*--- Multiply the Solution var with density to get the conservative transported quantity, if necessary. ---*/
         const su2double density = flowNodes->GetDensity(iPoint);
-        const su2double density_old = compressible ? flowNodes->GetSolution_Old(iPoint, 0) : density;
-
+        
         for (unsigned short iVar = 0; iVar < nVar; iVar++) {
```

Comment:
Can you add some comments above this line to document why density old is not used please?
Just something about the mixed time level and a link to this pull request.